### PR TITLE
[13.0][ADD] sale_partner_selectable_option: This module limits the partners to be selected in the sales orders.

### PR DESCRIPTION
**Pull request by victoralmau**
This module limits the partners to be selected in the sales orders according to the **Selectable in sale orders** field.

Please @carlosdauden and @pedrobaeza can you review it?

@Tecnativa TT32528

### CODE REVIEW

File: sale_partner_selectable_option/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to **Contacts > Contacts**
+#. Create a record and uncheck the "Selectable in sale orders" box on the Sales tab.
```
Comment (pedrobaeza) [inline sale_partner_selectable_option/README.rst]:
```suggestion
#. Create a record and uncheck the "Selectable in sales orders" box on the Sales tab.
```

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
> Sorry @HaraldPanten you are not allowed to merge.
> 
> To do so you must either have push permissions on the repository, or be a declared maintainer of all modified addons.
> 
> If you wish to adopt an addon and become it's [maintainer](https://odoo-community.org/page/maintainer-role), open a pull request to add your GitHub login to the `maintainers` key of its manifest.

@pedrobaeza 🤔?

**Comment by pedrobaeza** [comment]
Not sure why you don't have permissions. @sbidoul ?

**Pull request merged**